server/plugin/client.go

```diff
@@ -192,6 +192,15 @@ func (c *client) call(basePath, method, path, contentType string, mattermostUser
 		path = baseURL.String() + path
 	}
 
+	// Check refresh token only for APIs other than OAuth
+	if basePath != constants.BaseOauthURL {
+		if IsAccessTokenExpired, refreshToken := c.plugin.IsAccessTokenExpired(mattermostUserID); IsAccessTokenExpired {
```

Comment:
`IsAccessTokenExpired` is this corrected in other PR?

Reply:
yes this fixed in master